joystream-node-arm.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM rust:1.52.1-buster AS rust
```

Comment:
can we name this file `joystream-node-armv7.Dockerfile`

Reply:
Done